cogip/tools/monitor/socketiocontroller.py

```diff
@@ -24,8 +23,10 @@ class SocketioController(QtCore.QObject):
             Qt signal emitted to log messages in UI console
         signal_new_menu:
             Qt signal emitted to load a new menu
-        signal_new_robot_pose:
-            Qt signal emitted on robot pose update
+        signal_new_robot_pose_current:
+            Qt signal emitted on robot pose current update
+        signal_new_robot_order:
+            Qt signal emitted on robot pose order update
```

Comment:
This commit does more than just "remove unused import".
Should be splitted.

Reply:
Done.